test/using-object-values.html

```diff
@@ -259,6 +264,45 @@
           expect(combobox.inputElement.value).to.eql('not found');
         });
       });
+
+      describe('when item label and value paths are undefined', () => {
+        let error;
+        let comboboxUndefinedPaths;
+
+        const selectItemComboboxUndefinedPaths = (index) => {
+          // simulates clicking on the overlay items, but it more reliable in tests.
+          comboboxUndefinedPaths.$.overlay.dispatchEvent(new CustomEvent('selection-changed', {
+            detail: {item: comboboxUndefinedPaths.items[index]}
+          }));
+        };
+
+        beforeEach(() => {
+          error = sinon.stub(console, 'error');
```

Comment:
If you're stubbing `console.error` on `beforeEach`, I'd suggest restoring it in `afterEach`

Reply:
Or optionally do the stubbing in the test cases directly